### PR TITLE
fix: 🐛 directories structure for Snyk.VisualStudio.Extension.Shared [ROAD-652]

### PR DESCRIPTION
**Pull request by aldanchenko**
Updated extension id for proper value. And clear extension description.

### CODE REVIEW

File: Snyk.VisualStudio.Extension.2022/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-        <Description xml:space="preserve">The Visual Studio extension (Snyk’s Vulnerability Scanner) helps you find and fix security vulnerabilities in your projects. Within a few seconds, the extension will provide a list of all the different types of security vulnerabilities identified together with actionable fix advice. The extension combines the power of two Snyk products: Snyk Open Source and Snyk Code:
-
-1. With Snyk Open Source we find known vulnerabilities in both the direct and in-direct (transitive) open source dependencies you are pulling into the project.
-2. With Snyk Code we find known security vulnerabilities and code quality issues at a blazing speed looking at the code you and your team wrote.</Description>
+        <Description xml:space="preserve">The Visual Studio extension (Snyk’s Vulnerability Scanner) helps you find and fix security vulnerabilities in your projects. Within a few seconds, the extension will provide a list of all the different types of security vulnerabilities identified together with actionable fix advice. The extension combines the power of two Snyk products: Snyk Open Source and Snyk Code.</Description>
```
Comment (michelkaporin) [inline Snyk.VisualStudio.Extension.2022/source.extension.vsixmanifest:6]: Why description change?

**Review by michelkaporin — APPROVED** [review]

**Comment by aldanchenko** [comment]
@michelkaporin yesterday (before our solution with versions) I renamed namespace to `Snyk.VisualStudio.Extension.Shared`. If it's good idea we could merge it to main branch. Also if we want to use this namespace maybe I also need to add Shared folder to \\Snyk\\VisualStudio\\Extension\\?

**Comment by michelkaporin** [comment]
@aldanchenko Yes, please ensure rule "folders equal namespaces".

File: Snyk.VisualStudio.Extension.2022/Snyk.VisualStudio.Extension.2022.csproj [hunk 2]
```diff
@@ -13,7 +13,7 @@
     <ProjectGuid>{AB050A12-200E-48AE-898D-CB5532151A66}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Snyk.VisualStudio.Extension</RootNamespace>
+    <RootNamespace>Snyk.VisualStudio.Extension.Shared</RootNamespace>
```
Comment (michelkaporin) [inline Snyk.VisualStudio.Extension.2022/Snyk.VisualStudio.Extension.2022.csproj:16]: I am not sure if that's 100% right, can you perhaps explain this change? You have project named `Snyk.VisualStudio.Extension.2022` but the namespace is indicated as `...Shared`. And the assembly name is `Snyk.VisualStudio.Extension` - is it really correct, given we're in `...2022` project here?

Comment (aldanchenko) [reply Snyk.VisualStudio.Extension.2022/Snyk.VisualStudio.Extension.2022.csproj:16]: As I understand it reference to source code namespace. And we have root namespace in source code as Snyk.VisualStudio.Extension.Shared.

**Pull request merged**